tests/acceptance/features/apiGraph/enforcePasswordPublicLink.feature

```diff
@@ -0,0 +1,33 @@
+@api @env-config
+Feature: enforce password on public link
+  As a user
+  I want to enforce passwords on public links shared with upload, edit, or contribute permission
+  So that the password is required to make changes to the shared resources
```

Comment:
```suggestion
  I want to enforce passwords on public links shared with upload, edit, or contribute permission
  So that unauthorized people won't gain access to the contents in the link
```

Reply:
IMO, unauthorized could be misleading here. But I don't have any good sentences to suggest here :smile:

Reply:
Maybe `So that the password is required to access the contents in the link`? But the link can be with read-only permission that doesn't require password to be set

Reply:
I think `So that the password is required to access the contents in the link` is okay because the feature is for `upload, edit, or contribute permission` permission only
`I want to enforce passwords on public links shared with upload, edit, or contribute permission`